roles/ssh_hardening/vars/Amazon_2.yml

```diff
@@ -7,6 +7,7 @@ ssh_group: root
 ssh_host_keys_owner: root
 ssh_host_keys_group: ssh_keys
 ssh_host_keys_mode: "0600"
+ssh_host_pub_keys_mode: "0644"
```

Comment:
Should I instead move these into defaults so that people can override them with their own, perhaps more restrictive (e.g. `0444`) modes? It would be more maintainable and I can't see this being distro-specific.

Reply:
People can overide them anyway. so it's kind of irrelevant where they are. Keeping them here groups things that belong together, so I have no hard objections.

Reply:
Sorry, I guess I meant override them using inventory vars, which I think is not possible when using role vars (https://docs.ansible.com/ansible/devel/playbook_guide/playbooks_variables.html#understanding-variable-precedence) unless I'm misunderstanding that.

But in any case, as you mention below these tasks are mostly ticking some boxes in a checklist :grin: so if this is more consistent, that also works for me!